src/core/less/structure/block.less

```diff
@@ -0,0 +1,39 @@
+// Block styling
+// =============
+
+.block {
+    position: relative;
+    background-color: @background-color;
+
+    .block-inner {
+        max-width: @device-width-large;
+        margin: auto;
+        padding: @block-padding-top 0 @block-padding-bottom;
+        // reduce padding as screen width decreases
+        @media all and (max-width: (@device-width-large + 20px)) {
```

Comment:
do we remove these media queries?

they are horrid with the +20px etc

it could be argued that these are theme specific perhaps?